script.js

```diff
@@ -52,22 +62,22 @@ const setupRound = (wordPairs, pairRenderLimitIndex) => {
 
   // create the divs for the words
   while (lastUsedPairIndex < pairRenderLimitIndex) {
-    const word = document.createElement("div");
+    const kanji = document.createElement("div");
     const wordValue = wordPairs[lastUsedPairIndex][0];
 
-    word.classList.add(`box`);
-    word.innerHTML = wordValue;
+    kanji.classList.add(`box`);
```

Comment:
I would suggest using names like `left` and `right` or `column1` , `column2` or maybe `key` and `value` etc instead of kanji/hiragana. This way if you decide to add the ability to play in reverse (hiragana -> kanji) you don't have to rewrite all the code :D This goes also for function names etc.

This is formally known as decoupling of the business language (or domain language, such as "kanji") from UI language (such as "column")

We want to have the code as generic as possible, especially if this is a low cost (just a matter of decision). Useful framing to have in mind: "if I want to sell my code as a product, it should be able to suit multiple businesses" - maybe someone would like to use your code to learn French? etc etc